enterprise/views.py

```diff
@@ -597,6 +626,16 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_customer, co
             'price_text': self.context_data['price_text'],
             'continue_link_text': self.context_data['continue_link_text'],
             'course_modes': filter_audit_course_modes(enterprise_customer, course_modes),
+            'course_effort': course_effort,
+            'level_text': self.context_data['level_text'],
+            'effort_text': self.context_data['effort_text'],
+            'duration_text': self.context_data['duration_text'],
+            'course_overview': course_details['overview'],
+            'organization_logo': organization_logo,
+            'organization_name': organization_name,
+            'duration': duration,
+            'course_level_type': course_run.get('level_type', ''),
```

Comment:
I believe there is a level type at the course level - this seems to be the only field we're pulling out of the course run right now so it would be nice to avoid that extra call.